nomad/eval_endpoint.go

```diff
@@ -229,6 +229,27 @@ func (e *Eval) Ack(args *structs.EvalAckRequest,
 	if err := e.srv.evalBroker.Ack(args.EvalID, args.Token); err != nil {
 		return err
 	}
+
+	const cancelDesc = "cancelled after more recent eval was processed"
```

Comment:
```suggestion
	const cancelDesc = "canceled after more recent eval was processed"
```

I know the const is spelled wrong, but its value is actually spelled right! We should probably just correct the spelling of `structs.EvalStatusCancelled` since we don't promise API stability of that package.